site/src/components/form.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Form
+description: The form related components in the Atlas Design System
+template: standard
+---
+
+# Form
+
+Forms contain form control elements such as the following:
+
+- [`.button`](./button.md)
+- [`.checkbox`](./checkbox.md)
+- [`.help`](./help.md)
+- [`.input`](./input.md)
+- [`.label`](./label.md)
+- [`.radio`](./radio.md)
+- [`.select`](./select.md)
+- [`.textarea`](./textarea.md)
+- custom form elements
+
+## Form Field
+
+The `.field` component is a container for form control elements to provide consistency.
+It can be used to hold `.field-body` containers, which holds the individual elements.
+`field-label` is used for the element's label.
+
+```html
+<fieldset class="field border-none">
+	<legend class="field-label label margin-bottom-xs" aria-describedby="drink-description">
+		Favorite drink
+	</legend>
+	<p class="field-description margin-bottom-xxs" id="drink-description">Which do you prefer?</p>
+	<div class="field-body">
+		<label class="checkbox margin-bottom-xxs">
```

Comment:
Thinking `.checkbox:not(:last-child)` add the margin rather than relying on folks to add utilities.

Reply:
Added checkbox rule